### ISSUE TITLE
by where i can start?

### ISSUE THREAD

**Issue by sancg** [ivanlamega/dbo-server#2]
Hi, i want to look this files and i wonder, what is the basics that we start with. I'm student of development thing.
Thanks if you can ask.

**Comment by ivanlamega** [comment]
The crashing game due to one more column in the database, eliminating that column the server works perfectly. Right now I do not remember what column it is, I have not touched this code for a long time.
Sorry for the delay in my response.

**Comment by sancg** [comment]
Thanks so much, i'll be investigating the database then and see what's
happening.

**Comment by ivanlamega** [comment]
Try running it in debug mode.

**Comment by JayAlex51** [comment]
Did you figure out which column was causing the crash?

**Comment by ivanlamega** [comment]
Maybe one of these days I publish the source code of Akcore which is more advanced

**Comment by JayAlex51** [comment]
I tried akcore, but with Akcore I can't log in at all. With your files I can log in, but cant get ingame. Im not sure which column in which table to deleted

**Comment by ivanlamega** [comment]
Anyway, this server has nothing, if you manage to enter you can only run through the main world and nothing else.

**Comment by JayAlex51** [comment]
Is there a TW akcore source? I only see Akcore KR out

**Comment by ivanlamega** [comment]
Yes, there is an akcore tw source code, but it is a private repository to which I have access.

**Comment by sancg** [comment]
I got an AKcore tw but i don't know if it works with this proyect. Here is
the link:  https://bitbucket.org/akcore/akcore/src
Btw it's old.

**Comment by ivanlamega** [comment]
that source is Korean, although it has a tw branch, I don't know if it has much progress, I have another one that nobody made known

**Comment by sancg** [comment]
Oh ok. I understand that. I'll look into the files and i'll see what can i
made. I really want some functional server and learning a bit c ++. Thanks
again for the open channel with us. 🙏

**Comment by ivanlamega** [comment]
I'm on the cell phone, when I enter the computer, I will upload the repository here on GitHub

**Comment by JayAlex51** [comment]
Thanks a lot Ivan. That would be perfect. How much progress does the TW version have? Also @sancg I am doing the same. Would you like to team up?

**Comment by sancg** [comment]
That would be great and more helpful for to improve skills JayAlex. For
this project we need to start with same files, it’ll save us a lot of time
and headaches.

Contact me.
And again Ivan thanks

**Comment by JayAlex51** [comment]
That would be perfect. Add me on discord JayAlex#6293

**Comment by sancg** [comment]
SankDES #8911 Use mine i can't add you.

**Comment by JayAlex51** [comment]
Added you

**Comment by ivanlamega** [comment]
https://github.com/ivanlamega/Akcore-TW/tree/master

**Comment by JayAlex51** [comment]
Thanks a lot, bro

**Comment by JayAlex51** [comment]
Which client does akcore use? I got a black screen. I think it might be on my end

**Comment by ivanlamega** [comment]
Dbo tw 1.68

**Comment by JayAlex51** [comment]
![image](https://user-images.githubusercontent.com/53713541/62746553-e4f74280-ba15-11e9-9af6-c1ae2de2a14c.png)

Are you getting the black screen as well? I hear music and all just the black screen and no errors in console. On the DBOG client 1.69

**Comment by ivanlamega** [comment]
Yes, it happens to me too, run it in debug mode and set a breakpoint

**Comment by JayAlex51** [comment]
Where would you set the breakpoint for this issue? I'd imagine it would be one of the files in the CharServer folder or Gameserver

**Comment by ivanlamega** [comment]
I really do not remember, look at what server you have the black screen and see what package. You don't know how to program?

**Comment by JayAlex51** [comment]
I'm learning from this project. I was told that there's a packet that you have to send to enable and disable features

**Comment by ivanlamega** [comment]
I think there is commented code, I do not remember it well, but check, see which server is black and check the code.

**Comment by JayAlex51** [comment]
All is well so far. We are still learning, but It seems players cant see other players. I don't think I'm missing any dependencies. We appreciate your communication and updates
